src/_gettsim/transfers/wohngeld.py

```diff
@@ -66,29 +66,134 @@ def wohngeld_m_wthh(
     if not erwachsene_alle_rentner_hh and (
         wohngeld_vorrang_wthh or wohngeld_kinderzuschl_vorrang_wthh
     ):
-        out = _wohngeld_nach_vermög_check_m_wthh
+        out = wohngeld_anspruchshöhe_wthh
     else:
         out = 0.0
 
     return out
 
 
+def wohngeld_anspruchshöhe_wthh(
+    wohngeld_basisbetrag_m_wthh: float,
+    wohngeld_anspruchsbedingungen_erfüllt_wthh: bool,
+) -> float:
+    """Housing benefit after wealth and income check.
+
+    This target is used to calculate the actual Wohngeld of the Bedarfsgemeinschaften
+    that passed the priority check against ALG2 and Kinderzuschlag.
+
+    Parameters
+    ----------
+    wohngeld_basisbetrag_m_wthh
+        See :func:`wohngeld_basisbetrag_m_wthh`.
+    wohngeld_anspruchsbedingungen_erfüllt_wthh
+        See :func:`wohngeld_anspruchsbedingungen_erfüllt_wthh`.
+
+    Returns
+    -------
+
+    """
+    if wohngeld_anspruchsbedingungen_erfüllt_wthh:
+        out = wohngeld_basisbetrag_m_wthh
+    else:
+        out = 0.0
+
+    return out
+
+
+def wohngeld_anspruchshöhe_bg(
+    wohngeld_basisbetrag_m_bg: float,
```

Comment:
Remove the `wohngeld_basisbetrag` functions entirely and integrate them here. The point of the new structure is to only do the calculations if there actually is a claim on Wohngeld conditional on "absolute" checks like Vermögen, Mindesteinkommen.

Reply:
Sure, this will create a problem with the tests though because we didn't care about the Mindesteinkommen in the past. Many tests target what I had called "Basisbetrag", not the actual Anspruchshöhe.

I would add a `wohngeld_mindesteinkommen_erreicht_wthh=True` entry in all those tests and make a note at the top. If you prefer to rewrite all these tests such that they pass the Mindesteinkommen check (and are economically sensible), we can make an issue. I presume those are all regression tests.